### PR TITLE
support for shinyapps.io

### PR DESCRIPTION
**Pull request by mslynch**
This reverts commit 7b532ae97241ad1f64ade28fcf708dec9a45195d, reversing changes made to cfb7ce7f35a428a78fb1115d28a5b2de4c423bb8.

### CODE REVIEW

File: tests/test_api.py [hunk 1]
```diff
@@ -3,15 +3,15 @@
     require_api_key,
     require_connect,
 )
-from rsconnect.api import RSConnect, RSConnectExecutor, _to_server_check_list
+from rsconnect.api import RSConnectClient, RSConnectExecutor, _to_server_check_list
```
Comment (kgartland-rstudio) [inline tests/test_api.py:6]: Looks like this change breaks the `rsconnect-jupyter` plugin. `rsconnect-jupyter` tries to import `RSConnect` so we hit this error when installing/enabling `rsconnect-jupyter`:

```
ImportError: cannot import name 'RSConnect' from 'rsconnect.api'
```

Comment (bcwu) [reply tests/test_api.py:6]: For now I think we can just set `RSConnect = RSConnectClient` in api to fix this.

Comment (kgartland-rstudio) [reply tests/test_api.py:6]: Confirmed this has been fixed.  Still validating remaining workflows, will update when testing is complete.

**Comment by kgartland-rstudio** [comment]
rsconnect-python  version 1.9.1.dev80+g5d1680b

Kind of a nit pick but specifying a server where Connect doesn't exist now returns the html of the page in the error message.

Example:
`rsconnect add -n bad-test -s http://google.com -k o8Xyw657fMe6QOjTCDIVjDgnRz4HD9S`

```
Checking http://google.com...                    [ERROR]
Error:     http://google.com - failed to verify as RStudio Connect (Received an unexpected response from RStudio Connect (calling /__api__/server_settings): 404 Not Found
<!DOCTYPE html>
<html lang=en>
  <meta charset=utf-8>
  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">
  <title>Error 404 (Not Found)!!1</title>
  <style>
    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}#logo{background:url(//www.google.com/images/branding/googlelogo/1x/googlelogo_color_150x54dp.png) no-repeat;margin-left:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat;-webkit-background-size:100% 100%}}#logo{display:inline-block;height:54px;width:150px}
  </style>
  <a href=//www.google.com/><span id=logo aria-label=Google></span></a>
  <p><b>404.</b> <ins>That’s an error.</ins>
  <p>The requested URL <code>/__api__/server_settings</code> was not found on this server.  <ins>That’s all we know.</ins>).
    Do you need to use "https://google.com"?
```


On 1.9.0 build the same call returns:
```
Checking http://google.com...                    [ERROR]
Error:     http://google.com - failed to verify as RStudio Connect (Received an unexpected response from RStudio Connect: 404 Not Found).
    Do you need to use "https://google.com"?
```

**Comment by kgartland-rstudio** [comment]
Testing complete.  This is good to merge now.

Tested
- [x] adding removing updating server via rsconnect-python
- [x]  building, enabling, deploying from rsconnect-jupyter
- [x] adding, removing, updating server via rsconnect-jupyter
- [x] ran rsconnect-python integration tests (deploy, write-manifest, cypress tests) against this branch

**Pull request merged**